test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -1,21 +1,65 @@
 import ava from 'ava';
```

Comment:
Glad you combined all these.  It was a pain in the ass to have to update each file.  We had good intentions, it's just become impractical

Reply:
yip! :+1: